File: C/Developer/Tutorial/Learn/While Loop.c
#include <stdio.h>

int main(){
    // Variables
    int x = 1; int y = 10;

    // While Loop
    while(x <= y){
        if(x == y){
            printf("%d\n", x);
            // Break The Loop
            break;
        }
        printf("%d\t", x);
        x++;
    }

    // Do While Loop
    do{
        // Do Something Even If It's False Once
        if(x == y){
            printf("%d", x);
            // Break The Loop
            break;
        }
        printf("%d\t", x);
        x++;
    } while(x <= y);

    return 0;
}

File: Python/Developer/Tutorial/Learn/While Loop.py
#!/usr/bin/python3

# Variables
x, y = 0, 10

# While Loop: While
while x < y:
   print(x)
   x += 1

# While Loop: And
while x < y and x == 0:
   print(x)
   x += 1

# While Loop: And
while x < y or y > x:
   print(x)
   x += 1

# While Loop: Multiple Conditions
while (x < y and x == 0) or y > x:
   print(x)
   x += 1
